src/modules/scraper/adapter/messaging/MerkleDistributorBlocksEventsConsumerV2.ts

```diff
@@ -35,7 +35,7 @@ export class MerkleDistributorBlocksEventsConsumerV2 {
     const claimEvents = await this.getClaimEvents(filteredDistributorContracts, from, to);
     this.logNumberOfEventsFound(claimEvents, from, to);
     const insertResults = await this.insertClaimEvents(chainId, claimEvents);
-    const messages = insertResults.map((claim) => ({claimId: claim.id}))    
+    const messages = insertResults.map((claim) => ({claimId: claim.id}));    
```

Comment:
hmm how did these get through CI? 🤔

Reply:
It is related to [this config](https://github.com/across-protocol/scraper-api/pull/465/files/16f383a006d6fe15bd210b4950d1db9b7a8152fc#diff-9601a8f6c734c2001be34a2361f76946d19a39a709b5e8c624a2a5a0aade05f2R54) we just added missing